Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl

```diff
@@ -0,0 +1,253 @@
+          "\\_SB.PC00.I2C0", 0x00, ResourceConsumer, DEV0,) })
+          CreateWordField (IIC0, DEV0._ADR, DAD0)
+          Store (Arg0, DAD0)
+          Return (IIC0) }
+        Case(1) { Name (IIC1 , ResourceTemplate () { I2cSerialBus (0, ControllerInitiated, 400000, AddressingMode7Bit,
+          "\\_SB.PC00.I2C1", 0x00, ResourceConsumer, DEV1,) })
+          CreateWordField (IIC1, DEV1._ADR, DAD1)
+          Store (Arg0, DAD1)
+          Return (IIC1) }
+        Case(2) { Name (IIC2 , ResourceTemplate () { I2cSerialBus (0, ControllerInitiated, 400000, AddressingMode7Bit,
+          "\\_SB.PC00.I2C2", 0x00, ResourceConsumer, DEV2,) })
+          CreateWordField (IIC2, DEV2._ADR, DAD2)
+          Store (Arg0, DAD2)
+          Return (IIC2) }
+        Case(3) { Name (IIC3 , ResourceTemplate () { I2cSerialBus (0, ControllerInitiated, 400000, AddressingMode7Bit,
+          "\\_SB.PC00.I2C3", 0x00, ResourceConsumer, DEV3,) })
+          CreateWordField (IIC3, DEV3._ADR, DAD3)
+          Store (Arg0, DAD3)
+          Return (IIC3) }
+        Case(4) { Name (IIC4 , ResourceTemplate () { I2cSerialBus (0, ControllerInitiated, 400000, AddressingMode7Bit,
+          "\\_SB.PC00.I2C4", 0x00, ResourceConsumer, DEV4,) })
+          CreateWordField (IIC4, DEV4._ADR, DAD4)
+          Store (Arg0, DAD4)
+          Return (IIC4) }
+        Case(5) { Name (IIC5 , ResourceTemplate () { I2cSerialBus (0, ControllerInitiated, 400000, AddressingMode7Bit,
+          "\\_SB.PC00.I2C5", 0x00, ResourceConsumer, DEV5,) })
+          CreateWordField (IIC5, DEV5._ADR, DAD5)
+          Store (Arg0, DAD5)
+          Return (IIC5) }
+        Default {Return (0)}
+      }
+    }
+
+    //
+    // Create Interrupt Resource descriptor for _CRS usage
+    // Arg0 - GPIO Pad used as Interrupt source
+    // Arg1 - Edge (1) or Level (0) interrupt triggering
+    // Arg2 - Active Level: High (1) or Low (0)
+    // Returns buffer with 'Interrupt' resource descriptor
+    //
+    Method (INTB, 3, Serialized) {
+      Name (INTR, ResourceTemplate () { Interrupt (ResourceConsumer, Level, ActiveLow, ExclusiveAndWake,,,INTI) {0} })
+      // Update APIC Interrupt descriptor
+      CreateDWordField (INTR, INTI._INT, NUMI) // Interrupt Number
+      Store (INUM(Arg0), NUMI)
+      CreateByteField (INTR, INTI._HE, LEVI) // Level or Edge
+      Store (Arg1, LEVI)
+      CreateByteField (INTR, INTI._LL, ACTI) // Active High or Low
+      Store (Arg2, ACTI)
+
+      Return (INTR)
+    }
+
+  } // END Scope(\_SB.PC00)
+
+  Include ("Pch.asl")  // Not in this package. Refer to the PCH Reference Code accordingly
+  Include ("LpcB.asl")
+  Include ("Platform.asl")
+  Include ("Cpu.asl")
+  Include ("PciDrc.asl")
+  Include ("Video.asl")
+  Include ("PlatformGnvs.asl")
+  Include ("FwuWmi.asl")
+  Include ("Gpe.asl")
+
+  Include ("PchRpPxsxWrapper.asl")
+  Include ("WifiDynamicSar.asl")
+  Include ("Pep.asl")

```

Comment:
I remember you cleaned up some asl code, please make sure you uploaded the cleaned up one.

Reply:
Done